packages/react/src/evaluation/use-feature-flag.ts

```diff
@@ -31,6 +31,9 @@ type ConstrainedFlagQuery<T> = FlagQuery<
           : JsonValue
 >;
 
+// suspense options removed for the useSuspenseFlag hooks
```

Comment:
Remove suspense from the options of the `useSuspenseFlag` hook (it's always on).

Reply:
Could the `suspendWhileReconciling` still be useful to control?

Reply:
I think it's best to make the suspense hooks enable all suspense options by default. I think if more fine grained control is needed I'd use the 'useFlag' hook. I think in most cases , you either want all suspense or none.